### ISSUE TITLE
Cannot install with NPM when added as github dependency

### ISSUE THREAD

**Issue by lovesh** [paritytech/txwrapper#351]
For my JS project, I cannot use yarn and have to use NPM in my project and I wanted to specify `txwrapper` as a GitHub dependency but because of the `files` field in package.json, npm considers only the `lib` folder (for more check [here](https://stackoverflow.com/questions/56937044/why-does-yarn-install-checkout-a-github-repository-dependency-but-npm-install-do)). Is it possible to remove that field? 
For reference, i removed it [here](https://github.com/lovesh/txwrapper/commit/23ef17896244db162f502cf72335d17051fdcd61) and then specified it as GH dependency and a `postinstall` script in my package.json like below (I see its not cross platform) and I am able to use it.
```
"scripts": {
    .......,
    "postinstall": "cd node_modules/@substrate/txwrapper && npm run build && cd -"
},

"dependencies": {
     "@substrate/txwrapper": "github:lovesh/txwrapper#with-npm",
....
}
```

**Comment by emostov** [comment]
@lovesh can you elaborate why you want to install source code from github and not use the npm package?

I see the issue you are having, but in normal circumstances we don't expect users to transpile TS to JS  so trimming the package by shipping just the JS is reasonable.

**Comment by lovesh** [comment]
I wanted the package with `@polkadot/api` version 3.1.1 as a dependency which hasn't been released yet

**Comment by emostov** [comment]
Gotchya, I will make a release soon. 

Unfortunate that npm doesn't have resolution support the way yarn does.

**Comment by emostov** [comment]
Took at bumping polkadot-js/api and seems like there are some breaking changes to internals of metadata. Will try and update in the next 24 hours

**Comment by lovesh** [comment]
Thank you @emostov

**Comment by emostov** [comment]
This PR has @polkadot/api 3.2.2: https://github.com/paritytech/txwrapper/pull/352

**Comment by lovesh** [comment]
Thanks @emostov

**Comment by emostov** [comment]
@lovesh & @EvanXzj also just published a version with @polkadot/api 3.1.1 to npm
